src/PresentationalComponents/Filters/DropdownFilters.js

```diff
@@ -0,0 +1,104 @@
+import React, { Component } from 'react';
+import { Dropdown, DropdownToggle } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+import CheckboxFilter from './CheckboxFilter.js';
+import { RULE_CATEGORIES } from '../../AppConstants';
+import './_dropdown.scss';
+
+class DropdownFilters extends Component {
+    state = {
+        isOpen: false
+    };
+
+    addRemoveFilters = (selectedValue, filterName, isChecked) => {
+        isChecked ? this.props.addFilter(filterName, selectedValue) : this.props.removeFilter(filterName, selectedValue);
+    };
+
+    onToggle = isOpen => {
+        this.setState({
+            isOpen
+        });
+    };
+
+    render () {
+        const { hideCategories, filters } = this.props;
+        const { isOpen } = this.state;
+        const filterData = [
```

Comment:
Can you move this const to special file or at least above component definiton? This way we will not recreate these values everytime we re re-render component.